subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_7.adoc

```diff
@@ -731,6 +731,12 @@ Its functionality has been merged into the `gradlePlugin` block.
 These changes require recent versions of the Plugin Publish plugin (link:https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.1.0[1.0.+]).
 Documentation on configuring plugin publication can be found both link:https://plugins.gradle.org/docs/publish-plugin[on the Portal] and <<publishing_gradle_plugins#configure_the_plugin_publishing_plugin,in the user manual>>.
 
+==== Upgrade of the version catalog parser
```

Comment:
❌ This should land to 8.x upgrade guide, right?

Reply:
🤦 Right, fixing

That's the part I forgot to move when reviving this PR

Reply:
Fixed